README.md

```diff
@@ -150,11 +150,10 @@ const { lastIssues } = await graphql(`query lastIssues($owner: String!, $repo: S
     }
   }`, {
     owner: 'octokit',
-    repo: 'graphql.js'
+    repo: 'graphql.js',
     headers: {
       authorization: `token secret123`
     }
-  }
```

Comment:
I think this bracket needs to remain

Reply:
That one was causing an error as well. The matching bracket is as the following screenshot.
<img width="940" alt="Screen Shot 2020-08-14 at 16 01 57" src="https://user-images.githubusercontent.com/7040242/90222811-c6f1c880-de47-11ea-8c67-d953b6e7550e.png">

Reply:
I see, thank you!